productizer/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "example-productizer-httpsig"
+version = "0.1.0"
+description = "Example Productizer supporing HTTP Message Signatures"
+authors = ["IOXIO"]
+packages = [{include = "app", from = "."}]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+
+flask = "^2.2.3"
+pydantic = "^1.10.6"
```

Comment:
Could maybe consider updating to latest version of pydantic and use it with the `dotenv` extras and get rid of the extra direct dependency on `python-dotenv` to avoid version mismatches later on.